src/libponyc/pass/refer.c

```diff
@@ -702,7 +702,9 @@ static bool qualify_typeref(pass_opt_t* opt, ast_t* ast)
   ast_setdata(ast, (void*)def);
   ast_setid(ast, TK_TYPEREF);
 
-  pony_assert(typeref == ast_pop(ast));
+  ast_t* popped = ast_pop(ast);
+  (void)popped;
+  pony_assert(typeref == popped);
```

Comment:
The name doesn't matter much, but since many things are popped here, one suggested name is `first_child`.

```suggestion
  ast_t* first_child = ast_pop(ast);
  (void)first_child;
  pony_assert(typeref == first_child);
```